### PR TITLE
feat: add `plop-all` command

### PR DESCRIPTION
**Pull request by sebamarynissen**
This PR adds a `plop-all` command which can be used to plop all lots from a folder on a city tile. We've also refactored a bunch of the code so that we now read in the installation, plugins and regions folder the first time the command is run.

### CODE REVIEW

**Comment by sebamarynissen** [comment]
A note on the installation folder: perhaps that we shouldn't make this a hard requirement, as people might run some commands without having SimCity 4 installed.

File: lib/cli/setup-env.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+			message: `Locate your SimCity 4 ${name} folder`,
+			type: 'directory',
+			filter: info => info.isDirectory(),
+		});
+	}
+	return folder;
+}
+
+// # isValidInstallation(folder)
+async function isValidInstallation(folder) {
```
Comment (sebamarynissen) [inline lib/cli/setup-env.js:95]: We need to allow running commands without a valid installation.

**Comment by Zasco** [comment]
> A note on the installation folder: perhaps that we shouldn't make this a hard requirement, as people might run some commands without having SimCity 4 installed.

New issue?

**Comment by sebamarynissen** [comment]
What do you mean?

**Comment by Zasco** [comment]
> What do you mean?

Should we/I open an issue for that feature?

**Comment by sebamarynissen** [comment]
No, I prefer not opening issues for stuff that's already known and mentioned in a PR. Issues are better suited for stuff that comes up while during usage.

**Pull request merged**